src/okteto.ts

```diff
@@ -276,25 +279,21 @@ export function getLanguages(): RuntimeItem[] {
 
 }
 
-export function init(manifestPath: vscode.Uri, choice: string): boolean {
+export async function init(manifestPath: vscode.Uri, choice: string) {
   try{
-    const r = execa.commandSync(`${getBinary()} init --overwrite --file=${manifestPath.fsPath}`, {
+    const r = await execa.command(`${getBinary()} init --overwrite --file=${manifestPath.fsPath}`, {
       cwd: path.dirname(manifestPath.fsPath),
       env: {
-      "OKTETO_ORIGIN":"vscode",
-      "OKTETO_LANGUAGE":choice
-      } 
-    });
-    
+        "OKTETO_ORIGIN":"vscode",
+        "OKTETO_LANGUAGE":choice
+        } 
+      });
     if (r.failed) {
-      return false;
+      throw new Error(`okteto init failed: ${r.stdout}`);
```

Comment:
This will generate a double error message: `okteto init failed: okteto init failed: xxxx`, right?

Reply:
`r.stdout` is the output of the command. It can be anything from `command not found` to some other errors. Do you think we should just raise the error with r.stdout directly?

Reply:
fixed it.